sites/fast-website/src/app/svg/icon-brand.svg

```diff
@@ -0,0 +1,26 @@
+<svg width="114" height="40" viewBox="0 0 114 40" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
We may need to optimize this SVG a little bit due to all of the IDs hanging out. If we're using it in more than one place, it could cause conflicts.

Reply:
I will check with Jason R, to get me a better-optimized SVG